CHANGELOG.rst

```diff
@@ -6,6 +6,13 @@ This document describes changes between each past release.
 3.0.2 (unreleased)
 ==================
 
+**Protocol**
+
+- Added the ``GET /contribute.json`` endpoint for open-source information (fixes #607)
+
+Protocol is now in version **1.6**. See `API changelog <http://kinto.readthedocs.io/en/latest/api/>`_.
```

Comment:
I would probably write that a protocol is "at" a version or just "is version 1.6". To me, "in version 1.6" sounds like it's contained in the version, like a feature of the version.